src/common/darktable.c

```diff
@@ -1722,6 +1722,19 @@ void dt_configure_runtime_performance(const int old, char *info)
     g_strlcat(info, "\n\n", DT_PERF_INFOSIZE);
   }
 
+  if(old < 6)
+  {
+    g_strlcat(info, INFO_HEADER, DT_PERF_INFOSIZE);
+    g_strlcat(info, _("some global config values relevant for OpenCL performance are not used any longer."), DT_PERF_INFOSIZE);
+    g_strlcat(info, "\n", DT_PERF_INFOSIZE);
+    g_strlcat(info, _("instead you will find 'per device' data in 'cldevice_canonical-name'. content is:"), DT_PERF_INFOSIZE);
+    g_strlcat(info, "\n  ", DT_PERF_INFOSIZE);
+    g_strlcat(info, _("  'avoid_atomics'  'micro_nap'  'pinned_memory'  'clroundup'  -magic-"), DT_PERF_INFOSIZE);
+    g_strlcat(info, "\n  ", DT_PERF_INFOSIZE);
+    g_strlcat(info, _("you may tune as before except -magic-"), DT_PERF_INFOSIZE);
```

Comment:
Why is magic around minus sign whereas the others have simple-quote?

Reply:
good point, i meant this as a hint "don't tune"

Reply:
Well, I'll use the same display format.

Reply:
ok, will fix

Reply:
done in latest commit